### ISSUE TITLE
Make parameter files parsing optional

### ISSUE THREAD

**Issue by aprokop** [open-research/sumatra#300]
Sumatra 0.7.3

I have a script that manages parameter files in the form "param=value"
and creates a new parameter file based on the original and some extra input.
The important part was that there were no spaces around =. Sumatra did the transformation resulting in adding quote marks around parameters and spaces around =. 

I looked around for a way to make Sumatra completely ignore the file, but could not figure things out. I had to hack installation of sumatra the following way:

```
--- commands.py.orig    2015-10-01 17:16:39.668133745 -0600
+++ commands.py 2015-10-01 17:15:17.099492531 -0600
@@ -73,7 +73,7 @@
     for arg in args:
         have_parameters = False
         if os.path.isfile(arg):  # could be a parameter file or a data file
-            parameters = build_parameters(arg)
+            parameters = None
             if parameters is not None:
                 parameter_sets.append(parameters)
                 script_args.append("<parameters>")
```

**Comment by apdavison** [comment]
Sorry for the long delay in replying.

If you have a custom parameter file format, the best approach is write your own `ParameterSet` subclass and register it using the plugin mechanism - see http://sumatra.readthedocs.org/en/0.7.3/plugins.html
Let me know if you need help with this.

However, I agree it would be helpful to be able to turn off parameter file parsing, so I will leave this ticket open. Perhaps an `--ignore-parameters` option for `smt configure`?

**Comment by aprokop** [comment]
> Perhaps an --ignore-parameters option for smt configure?

Yes, it would be ideal from the user perspective, as it requires minimal adjustment.

**Comment by apdavison** [comment]
Implemented in #373
